### PR TITLE
refactor: replace verb renovating with plain English

### PR DESCRIPTION
**Pull request by HonkingGoose**
@coderabbitai: ignore

## Changes

- Replace verb `renovating` with plain English

## Context

Two maintainers gave my rewrite idea a thumbs up in this Discussion:

- #23405

I'm not closing any issue with this PR.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/workers/repository/dependency-dashboard.ts [hunk 1]
```diff
@@ -175,7 +175,7 @@ function appendRepoProblems(config: RenovateConfig, issueBody: string): string {
     );
     newIssueBody += '## Repository problems\n\n';
     newIssueBody +=
-      'These problems occurred while renovating this repository.\n\n';
+      'Renovate tried to update this repository, but found these problems.\n\n';
```
Comment (HonkingGoose) [inline lib/workers/repository/dependency-dashboard.ts:178]: By the way, some users search the logs for specific strings to build out their own dashboard:

- #23105

So do we consider a log message change a `patch` or `refactor` type thing then? This `refactor` may be a _breaking change_ for some of those "building my own telemetry for Renovate"-users. 🙃

File: lib/workers/repository/dependency-dashboard.ts [hunk 2]
```diff
@@ -176,7 +176,7 @@ function appendRepoProblems(config: RenovateConfig, issueBody: string): string {
     newIssueBody += '## Repository problems\n\n';
     const repoProblemsHeader =
       config.customizeDashboard?.['repoProblemsHeader'] ??
-      'These problems occurred while renovating this repository.';
+      'Renovate tried to update this repository, but found these problems.';
```
Comment (HonkingGoose) [inline lib/workers/repository/dependency-dashboard.ts:179]: By the way, some users search the logs for specific strings to build out their own dashboard:

- #23105

So do we consider a log message change a `patch` or `refactor` type thing then? This `refactor` may be a _breaking change_ for some of those "building my own telemetry for Renovate"-users. 🙃

File: docs/usage/configuration-options.md [hunk 3]
```diff
@@ -25,7 +25,8 @@ You can store your Renovate configuration file in one of these locations:
 !!! warning
```
[the remark this answers is not in the document]
Comment (rarkins) [reply docs/usage/configuration-options.md:25]: `'Restarting repository job after automerge result'`

File: docs/usage/configuration-options.md [hunk 4]
```diff
@@ -25,7 +25,8 @@ You can store your Renovate configuration file in one of these locations:
...
     Storing the Renovate configuration in a `package.json` file is deprecated and support may be removed in the future.
 
-When renovating a repository, Renovate tries to detect the configuration files in the order listed above, and stops after the first one is found.
+When Renovate updates a repository, it tries to find the configuration files in the order listed above.
```
Comment (rarkins) [inline docs/usage/configuration-options.md:28]:
```suggestion
When Renovate runs on a repository, it tries to find the configuration files in the order listed above.
```

File: lib/workers/repository/dependency-dashboard.ts (hunk 2 above)
Comment (rarkins) [reply lib/workers/repository/dependency-dashboard.ts:179]: It's not always _updating_ a repository, and problems can happen before it gets to updating. So I'd prefer to replace "renovating" with "running on" or "processing"

**Review by rarkins — CHANGES_REQUESTED** [review]

File: lib/workers/repository/dependency-dashboard.ts (hunk 2 above)
Comment (HonkingGoose) [reply lib/workers/repository/dependency-dashboard.ts:179]:
```suggestion
      'Renovate tried to run on this repository, but found these problems.';
```

**Review by rarkins — APPROVED** [review]

**Pull request merged**